### ISSUE TITLE
When will support VS 1.13.0 or auto update

### ISSUE THREAD

**Issue by Andrewyg** [garethflowers/vscode-portable#25]

**Comment by dragonthegn** [comment]
+1

**Comment by garethflowers** [comment]
Auto-updating is non-trivial task - but I will investigate.

In the meantime I've released [v1.14.2](011338f3099fc1bba3d0377cdf73b7a7).

**Comment by garethflowers** [comment]
Duplicate of #5

**Comment by alekk84** [comment]
how about v1.17?

**Comment by RoyiAvital** [comment]
Version 1.20 is out.

By the way, is there a reason why I can't disable Git Integration with this version?
I copied all needed settings to my settings as described here https://stackoverflow.com/questions/30331338.
Yet each time I launch the program it searches for Git and notifies it can't be found.

I'm using the latest current version ([1.19.2](https://github.com/garethflowers/vscode-portable/releases/tag/v1.19.2)).

Thank You.

**Comment by garethflowers** [comment]
Having a separate installer that always downloads the latest version is probably a good move. Let me check it out.

**Comment by garethflowers** [comment]
@RoyiAvital give [this release](https://github.com/garethflowers/vscode-portable/releases/tag/Latest) a try.

**Comment by RoyiAvital** [comment]
I will give it a try.
What's needed is someone watching they don't change what they write on Registry, locations of files, etc... on newer version so it stays Portable.

By the way, do you change all system variables before launching or do you use Copy on exit (I hope I understand correctly how PortableApps work)?

**Comment by RoyiAvital** [comment]
Hi,
Do you use the `--user-data-dir` and `--extension-dir` launcher properties?

See here - https://github.com/Microsoft/vscode/issues/329#issuecomment-385172447.

**Comment by vanja-san** [comment]
As far as my know, they do not use it there. It's PortableApps, it there own resources.

**Comment by garethflowers** [comment]
Yes - we pass both as command line arguments to the code executable.

https://github.com/garethflowers/vscode-portable/blob/5ee80741d7ec551872e009d072852b6971134723/VSCodePortable/App/AppInfo/Launcher/VSCodePortable.ini#L4

**Comment by garethflowers** [comment]
Just noticed it looks we are using the wrong argument for the extensions though. I'll get that sorted.

**Comment by RoyiAvital** [comment]
@garethflowers ,
I think you also should update the Online Installer which always download the latest version with this fix.

**Comment by RoyiAvital** [comment]
@garethflowers ,
Any update on the Online Installer (The one which downloads the latest)?
Could you fix the directory issue on it as well?

Thank You.

**Comment by RoyiAvital** [comment]
@garethflowers ,
Any update on the Online Installer (The one which downloads the latest)?
Could you fix the directory issue on it as well?

Thank You.

**Comment by RoyiAvital** [comment]
@garethflowers ,
Are you aware of that - https://github.com/Microsoft/vscode/issues/329#issuecomment-398144113?

Could you also update the template of the "Always Latest" version?
